src/lib/Libwin/passwd.c

```diff
@@ -4170,7 +3910,9 @@ wsystem(char *cmdline, HANDLE user_handle)
 	} else {
 		flags = flags|CREATE_UNICODE_ENVIRONMENT;
 		if (!CreateEnvironmentBlock(&user_env, user_handle, FALSE)) {
+			run_exit = GetLastError();
```

Comment:
why do we need this line? This is being done later anyways

Reply:
besides log_errf() could automatically log it?

Reply:
This part of code was added via [reference](https://github.com/openpbs/openpbs/pull/1770/commits/196a3dff8fe7dcbd4cdb597a8bd7af6ba8ccc0fa) which by mistake got deleted via [here](https://github.com/openpbs/openpbs/pull/1781/files#diff-60063bfd099fee0809396e4aebb54f6aL4151)